cmd/askd/handlers/form_submission.go

```diff
@@ -253,3 +307,25 @@ func (formSubmissionHandle) RemoveFlag(c *app.Context) error {
 
 	return nil
 }
+
+// encodeCSV gets all the submissions and encode them into a CSV
+func encodeCSV(s []submission.Submission) ([]byte, error) {
+
+	var buf bytes.Buffer
+	w := csv.NewWriter(&buf)
+	if err := w.Write(s[0].GetQuestions()); err != nil {
```

Comment:
Other way to do it to resolve that is to convert it into other structure first ( a slice of dictionaries) and then into the CSV format.